scripts/ci/gitlab/pipeline/build.yml

```diff
@@ -15,6 +15,7 @@
   variables:
     RUSTFLAGS: "-D warnings"
   script:
+    - cargo install --locked --git https://github.com/paritytech/try-runtime-cli --rev a93c9b5abe5d31a4cf1936204f7e5c489184b521
```

Comment:
This is a really bad idea. I already see this breaking. Why is the CLI not just part of the repo and just its own binary?

Reply:
Motivations for an independent project are listed here: https://github.com/paritytech/substrate/issues/12975

What're the biggest breakage risks you see?

Reply:
That is maybe breaks the CI, but I don't want to block on this here.